docs/pages/index.mdx

```diff
@@ -102,10 +102,17 @@ Get started with Teleport Identity:
 ### Teleport Policy
 
 **Teleport Policy** unifies and controls access policies across all your
-infrastructure. With Teleport Policy’s Access Graph feature, you gain insights into role-based
-access control policies within Teleport and your cloud provider.
-
-Get started with [Teleport Policy](admin-guides/teleport-policy/teleport-policy.mdx).
+infrastructure. With Teleport Policy’s Access Graph feature, you gain insights
+into role-based access control policies within Teleport and your cloud provider.
```

Comment:
```suggestion
into role-based access control policies within Teleport, critical integrations and your cloud providers.
```

Reply:
Edited, though I used "critical infrastructure", since it's not the integrations you would pull policies from, but the tools etc. that we integrate with.